proto/osmosis/epochs/genesis.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package osmosis.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/epochs/types";
+
+message Epoch {
```

Comment:
Should this have a different name?  Maybe like `EpochSaga` or something?  Cause calling this an epoch is weird, because I think an epoch refers to one of time segments, not the whole metadata itself

Reply:
Modified to EpochInfo